packages/core/src/components/document-component/document-component.tsx

```diff
@@ -257,8 +266,36 @@ export class DocumentComponent implements ComponentInterface {
     );
   }
 
+  private parseWijzigactie(
+    input?: DocumentComponentInputType,
+  ): { tagName: string; wijzigactie: DocumentComponentWijzigactie | null } | undefined {
+    if (!input) {
+      return undefined;
+    }
+
+    let element: Element | null = null;
+
+    if (typeof input === "string") {
+      const parser = new DOMParser();
```

Comment:
Elke keer als deze functie wordt aangeroepen wordt er een nieuwe DOMParser gemaakt, is dat nodig?